parity/rpc.rs

```diff
@@ -180,7 +182,7 @@ pub fn new_ws<D: rpc_apis::Dependencies>(
     let signer_path;
     let path = match conf.support_token_api {
         true => {
-            signer_path = ::signer::codes_path(&conf.signer_path);
+            signer_path = crate::signer::codes_path(&conf.signer_path);
```

Comment:
Is this related to Prometheus?

Reply:
The answer is no, but it is just syntax change needed fo the rust 2018 edition.

Reply:
This is still valid with 2015